api/scpca_portal/management/commands/generate_computed_files.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import shutil
+from argparse import BooleanOptionalAction
+from concurrent.futures import ThreadPoolExecutor
+from functools import partial
+from threading import Lock
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from scpca_portal import common, s3, utils
+from scpca_portal.models import Project
+from scpca_portal.models.computed_file import ComputedFile
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler())
+
+
+class Command(BaseCommand):
+    help = """
+    Data files should be contained in an S3 bucket called scpca-portal-inputs.
+
+    The directory structure for this bucket should follow this pattern:
+        /project_metadata.csv
+        /SCPCP000001/libraries_metadata.csv
+        /SCPCP000001/samples_metadata.csv
+        /SCPCP000001/SCPCS000109/SCPCL000126_filtered.rds
+        /SCPCP000001/SCPCS000109/SCPCL000126_metadata.json
+        /SCPCP000001/SCPCS000109/SCPCL000126_processed.rds
+        /SCPCP000001/SCPCS000109/SCPCL000126_qc.html
+        /SCPCP000001/SCPCS000109/SCPCL000126_unfiltered.rds
+        /SCPCP000001/SCPCS000109/SCPCL000127_filtered.rds
+        /SCPCP000001/SCPCS000109/SCPCL000127_metadata.json
+        /SCPCP000001/SCPCS000109/SCPCL000127_processed.rds
+        /SCPCP000001/SCPCS000109/SCPCL000127_qc.html
+        /SCPCP000001/SCPCS000109/SCPCL000127_unfiltered.rds
+
+    The files will be zipped up and stats will be calculated for them.
+
+    If run locally the zipped ComputedFiles will be copied to the
+    "scpca-local-data" bucket.
+
+    If run in the cloud the zipped ComputedFiles files will be copied
+    to a stack-specific S3 bucket."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--clean-up-input-data",
+            action=BooleanOptionalAction,
+            type=bool,
+            default=settings.PRODUCTION,
+        )
+        parser.add_argument(
+            "--clean-up-output-data",
+            action=BooleanOptionalAction,
+            type=bool,
+            default=settings.PRODUCTION,
+        )
+        parser.add_argument("--max-workers", type=int, default=10)
+        parser.add_argument("--scpca-project-id", type=str)
+        parser.add_argument(
+            "--update-s3", action=BooleanOptionalAction, type=bool, default=settings.UPDATE_S3_DATA
+        )
+
+    def handle(self, *args, **kwargs):
+        self.generate_computed_files(**kwargs)
+
+    @staticmethod
+    def clean_up_input_data() -> None:
+        shutil.rmtree(common.INPUT_DATA_PATH, ignore_errors=True)
+
+    @staticmethod
+    def clean_up_output_data() -> None:
+        shutil.rmtree(common.OUTPUT_DATA_PATH, ignore_errors=True)
```

Comment:
Note that I updated these functions from their implementations in the `load_data` command.
Over there, they are implemented as follows:
```python
    @staticmethod
    def clean_up_input_data(project) -> None:
        shutil.rmtree(common.INPUT_DATA_PATH / project.scpca_id, ignore_errors=True)

    @staticmethod
    def clean_up_output_data() -> None:
        for path in Path(common.OUTPUT_DATA_PATH).glob("*"):
            path.unlink(missing_ok=True)
```

For code running on Batch instances, it made more sense to me to delete all input and output data directories (and their contents) upon completion of file computation. Thoughts?